### PR TITLE
FIX: Very Minor Bug

### PR DESCRIPTION
**Pull request by DuckyDoesCode**
If the player/unit was nil then then the function would be aborted and it wouldn't re-enable the give item button. Instead it would re-enable the give money button.

Side-note: Moved it from the beginning of the exit statement to the end because of consistency...

#### Changes proposed in this pull request: 
- Changing the Control IDC from 2001 to 2002

### CODE REVIEW

**Comment by DuckyDoesCode** [comment]
@SimZor Talking of consistency...

File: Altis_Life.Altis/core/pmenu/fn_giveItem.sqf [hunk 1]
```diff
@@ -26,12 +26,12 @@ if (isNull _unit) exitWith {ctrlShow[2002,true];};
 //A series of checks *ugh*
 if (!([_val] call TON_fnc_isnumber)) exitWith {hint localize "STR_NOTF_notNumberFormat";ctrlShow[2002,true];};
```
Comment (SimZor) [inline Altis_Life.Altis/core/pmenu/fn_giveItem.sqf]: Now that you opened this pull request already, why not make it a bit nicer.

- No need for the extra parenthesis around if condition
- You do not need the semicolon before curly bracket

File: Altis_Life.Altis/core/pmenu/fn_giveItem.sqf [hunk 2]
```diff
@@ -26,12 +26,12 @@ if (isNull _unit) exitWith {ctrlShow[2002,true];};
...
 if (parseNumber(_val) <= 0) exitWith {hint localize "STR_NOTF_enterAmountGive";ctrlShow[2002,true];};
```
Comment (SimZor) [inline Altis_Life.Altis/core/pmenu/fn_giveItem.sqf]: No need for () around _val, also no need for semi-colon before closing curly bracket.

File: Altis_Life.Altis/core/pmenu/fn_giveItem.sqf [hunk 3]
```diff
@@ -26,12 +26,12 @@ if (isNull _unit) exitWith {ctrlShow[2002,true];};
...
-if (isNil "_unit") exitWith {ctrlShow[2001,true]; hint localize "STR_NOTF_notWithinRange";};
+if (isNil "_unit") exitWith {hint localize "STR_NOTF_notWithinRange";ctrlShow[2002,true];};
```
Comment (SimZor) [inline Altis_Life.Altis/core/pmenu/fn_giveItem.sqf]: no need for semi-colon before closing curly bracket.

File: Altis_Life.Altis/core/pmenu/fn_giveItem.sqf [hunk 4]
```diff
@@ -26,12 +26,12 @@ if (isNull _unit) exitWith {ctrlShow[2002,true];};
...
 if (!([false,_item,(parseNumber _val)] call life_fnc_handleInv)) exitWith {hint localize "STR_NOTF_couldNotGive";ctrlShow[2002,true];};
```
Comment (SimZor) [inline Altis_Life.Altis/core/pmenu/fn_giveItem.sqf]: No need for brackets around condition

no need for semi-colon before closing curly bracket.

File: Altis_Life.Altis/core/pmenu/fn_giveItem.sqf [hunk 5]
```diff
@@ -26,12 +26,12 @@ if (isNull _unit) exitWith {ctrlShow[2002,true];};
...
 
 [_unit,_val,_item,player] remoteExecCall ["life_fnc_receiveItem",_unit];
 _type = M_CONFIG(getText,"VirtualItems",_item,"displayName");
 hint format [localize "STR_NOTF_youGaveItem",_unit getVariable ["realname",name _unit],_val,(localize _type)];
 [] call life_fnc_p_updateMenu;
 
-ctrlShow[2002,true];
\ No newline at end of file
+ctrlShow[2002,true];
```
Comment (SimZor) [inline Altis_Life.Altis/core/pmenu/fn_giveItem.sqf]: No need for semi-colon

File: Altis_Life.Altis/core/pmenu/fn_giveItem.sqf [hunk 6]
```diff
@@ -26,12 +26,12 @@ if (isNull _unit) exitWith {ctrlShow[2002,true];};
 //A series of checks *ugh*
 if (!([_val] call TON_fnc_isnumber)) exitWith {hint localize "STR_NOTF_notNumberFormat";ctrlShow[2002,true];};
 if (parseNumber(_val) <= 0) exitWith {hint localize "STR_NOTF_enterAmountGive";ctrlShow[2002,true];};
-if (isNil "_unit") exitWith {ctrlShow[2001,true]; hint localize "STR_NOTF_notWithinRange";};
+if (isNil "_unit") exitWith {hint localize "STR_NOTF_notWithinRange";ctrlShow[2002,true];};
 if (!([false,_item,(parseNumber _val)] call life_fnc_handleInv)) exitWith {hint localize "STR_NOTF_couldNotGive";ctrlShow[2002,true];};
 
 [_unit,_val,_item,player] remoteExecCall ["life_fnc_receiveItem",_unit];
 _type = M_CONFIG(getText,"VirtualItems",_item,"displayName");
 hint format [localize "STR_NOTF_youGaveItem",_unit getVariable ["realname",name _unit],_val,(localize _type)];
```
Comment (SimZor) [inline Altis_Life.Altis/core/pmenu/fn_giveItem.sqf]: no need for the parenthesis

**Pull request merged**